src/App.tsx

```diff
@@ -17,8 +17,48 @@ import FourOhFour from "./pages/404";
 
 GoogleAnalytics.initialize(process.env.GA_TRACKING_CODE);
 
+const seoImage = process.env.DEFAULT_SEO_IMAGE;

```

Comment:
😍

Reply:
I pretty much just moved this out/up from https://github.com/onflow/flow-playground/pull/66/files#diff-46ef2374da6211486f3a408db68b2bcbL21